external_auth/views.py

```diff
@@ -0,0 +1,60 @@
+from __future__ import unicode_literals
+
+from django.contrib.auth.models import User
+from django.contrib import messages
+from external_auth.forms import PakwheelsLoginForm
+from external_auth.pakwheels_scraper import PakwheelsScraper
+from external_auth.pakwheels_profile import PakwheelsProfile
+from django.shortcuts import render
+from django.views.decorators.http import require_http_methods
+
+
+@require_http_methods(["GET", "POST"])
+def pakwheels_registration(request):
+    if request.method == 'GET':
+        pakwheels_login_form = PakwheelsLoginForm()
+        context = {'form': pakwheels_login_form}
+        return render(request, 'pakwheels_register.html', context)
+
+    pakwheels_login_form = PakwheelsLoginForm(request.POST)
+    username = pakwheels_login_form.data['username']
```

Comment:
are you sure this will be a username not the email address? How we are handling email and username while creating a user on our end.

Reply:
It can be both on the Pakwheels site. While creation, the username and email are associated with the default Django User object after profile is imported from Pakwheels.

Reply:
I have made corrections. This will be an email address as we use email address to login to Pakwheels.